# origin: marcelo (BitKeeper)
# cset: 1.1482.2.4 (2.4) key=418a6777HQyW4FAHRd2zRqroXw2w_w
# inclusion: upstream
# description:   mcp: Fix proc_delete_inode oops bug correction typo
# revision date: Fri, 26 Nov 2004 16:58:09 +0900
#
# S rset: ChangeSet|1.1482.2.3..1.1482.2.4
# I rset: fs/proc/base.c|1.20..1.21
#
# This is a BitKeeper generated diff -Nru style patch.
#
# ChangeSet
#   2004/11/04 15:31:35-02:00 marcelo@logos.cnet 
#     mcp: Fix proc_delete_inode oops bug correction typo
# 
# fs/proc/base.c
#   2004/11/04 15:30:42-02:00 marcelo@logos.cnet +1 -1
#   Fix proc_delete_inode oops bug correction typo
# 
#
===== fs/proc/base.c 1.20 vs 1.21 =====
--- 1.20/fs/proc/base.c	2004-11-04 11:25:16 +09:00
+++ 1.21/fs/proc/base.c	2004-11-05 02:30:42 +09:00
@@ -780,7 +780,7 @@
 	return inode;
 
 out_unlock:
-	node->u.generic_ip = NULL;
+	inode->u.generic_ip = NULL;
 	iput(inode);
 	return NULL;
 }
